src/components/Navbar.jsx

```diff
@@ -1,24 +1,33 @@
 import React from "react";
 import { Container, Menu } from "semantic-ui-react";
+import LogIn from "./LogIn";
+import { connect } from "react-redux";
 import "../css/style.css";
 
-const Navbar = () => {
+const Navbar = props => {
+  let logInActions;
+  if (props.currentUser.isSignedIn === false) {
+    logInActions = <LogIn />;
+  } else {
+    logInActions = "";
+  }
+
```

Comment:
Maybe chance this if statement with using ternary instead

Reply:
Done!